mapper/mapper.go

```diff
@@ -182,7 +182,7 @@ func mapDataPage(page *model.SearchPage, respC *searchModels.SearchResponse, lan
 
 	page.Data.KeywordFilter = coreModel.CompactSearch{
 		ElementId: "keywords",
-		InputName: "q",
+		InputName: "kw-query",
 		Language:  lang,
 		Label: coreModel.Localisation{
 			LocaleKey: "SearchKeywords",
```

Comment:
I don't think this is the right way to go about this - the javascript is for example still updating the q parameter. 

Having tested it, I also don't think it's necessary. This fix was the one in the .tmpl file I think as there's an extra input there which would've been confusing things.

Reply:
Yes I changed this to kw-query more as a 'nice to have' as there are 4 other parameters all named "q" on the same page (one for main search in the title bar, one for filters, one for sorts, and one for pagination. Having changed this it was easier to discover the fix in the tmpl file because I could see that it was one of the parts that was still using "q". I can change it back to "q" if you prefer though?

Reply:
👍 ok that makes sense, thanks